src/shared/lib/analytics/client.tsx

```diff
@@ -1,25 +1,26 @@
 import { OpenPanelComponent, type PostEventPayload, useOpenPanel } from "@openpanel/nextjs";
 
+import { env } from "@/env";
+
 const isProd = process.env.NODE_ENV === "production";
 
 const AnalyticsProvider = () => (
   <OpenPanelComponent
-    clientId={process.env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID!}
+    clientId={env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID!}
     trackAttributes={true}
     trackOutgoingLinks={isProd}
     trackScreenViews={isProd}
   />
 );
```

Comment:
We can avoid the `!` at the end while making the full of in AnalyticsProvider instead of his parent

```tsx
const AnalyticsProvider = () => {
  if (!env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID) {
    return null;
  }

  return (
  <OpenPanelComponent
    clientId={env.NEXT_PUBLIC_OPENPANEL_CLIENT_ID}
    trackAttributes={true}
    trackOutgoingLinks={isProd}
    trackScreenViews={isProd}
  />
)};
```